src/components/CharacterList/CharacterList.tsx

```diff
@@ -11,9 +11,10 @@ const CharacterList = ({ characters }: CharacterListProps) => {
   return (
     <UnorderedList role='characters-list'>
       <SimpleGrid spacing='20px' columns={{ sm: 1, md: 3, lg: 4 }}>
-        {characters.map((character) => (
-          <CharacterItem key={character.id} character={character} />
-        ))}
+        {characters.map((characterElement) => {
+          const { id, ...character } = characterElement
+          return <CharacterItem key={id} character={character} />
+        })}
```

Comment:
I see how you killed the bug, but it seems somewhat hacky. Isn't it preventable to get an "empty" character with an id in the first place somehow?